.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+name: Tests & Linting
+
+on: push
+
+jobs:
+  run:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:        
+        php-versions: ['7.2', '7.3', '7.4']
+      fail-fast: false
+    name: PHP ${{ matrix.php-versions }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php-versions }}
+    
+    - name: Install dependencies
+      run: composer install --no-interaction
```

Comment:
You can speed up subsequent builds with actions/cache. A Composer-specific example is here 👍 https://github.com/actions/cache/blob/master/examples.md#php---composer

Reply:
Thanks @pezholio. I've implemented that and it looks like it's knocked ~5 seconds off each job 👍

Reply:
Whoop!